xml/decoders/Uhlenbrock_Tillig BR89_73xx5.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <?xml-stylesheet type="text/xsl" href="../XSLT/decoder.xsl"?>
-<!-- Copyright (C) JMRI 2020, 2022 All rights reserved -->
+<!-- Copyright (C) JMRI 2019 All rights reserved -->
```

Comment:
This file is renamed and changed. It doesn't look right. It seems that you wanted to copy this file and change the copy, but instead renamed the original file.

Reply:
I made a copy of xml/decoders/Uhlenbrock_73xx5.xml to the xml/decoders/Uhlenbrock_Tillig BR89_73xx5.xml.